Writerside/topics/Matchmaking.md

```diff
@@ -1,2 +1,252 @@
 # Matchmaking
 
+The Catena matchmaker is responsible for grouping players or parties together to form games or matches. This may be done based on data sent by clients or retrieved from other sources and rules configured in the matchmaking service. The matchmaker can support both peer-to-peer and dedicated server arrangements. Additionally, Catena decouples matchmaking from server assignment and fleet and capacity management; these tasks are handled by the [](Match-Broker.md).
```

Comment:
Minor Nitpick

```suggestion
The Catena matchmaker is responsible for grouping players or parties together to form games or matches. This may be done based on data sent by clients or retrieved from other sources and rules configured in the matchmaking service. The matchmaker can support both peer-to-peer and dedicated server arrangements. 

Catena decouples matchmaking from server assignment and fleet and capacity management; these tasks are handled by the [](Match-Broker.md).
```